src/plugins/FileService/constants.ts

```diff
@@ -0,0 +1,11 @@
+export const pluginName = 'File Service'
+export const identifier = 'com.maskbook.fileservice'
+export const pluginId = 'maskbook.fileservice'
+export const META_KEY_1 = 'com.maskbook.fileservice:1'
+
+export const MAX_FILE_SIZE = 0xa00000 // = 10 MiB
+
+export const landing = 'https://files.maskbook.com/partner/arweave/landing-page.html'
+// in production
+// https://service.maskbook.com/arweave-remote-signing
+export const signing = 'https://f8149vcdfi.execute-api.ap-east-1.amazonaws.com/arweave-remote-signing'
```

Comment:
will you change to the production url?

Reply:
PRODUCTION NOT READY,
Wait for @yisiliu